Doc/library/shutil.rst

```diff
@@ -680,6 +681,33 @@ The resulting archive contains:
     -rw-r--r-- tarek/staff   37192 2010-02-06 18:23:10 ./known_hosts
 
 
+.. _shutil-archiving-example-with-basedir:
```

Comment:
Could you please link to the example from the `make_archive()` documentation?

Reply:
Thanks @berkerpeksag for the review? From where exactly would you think it would be best to link? From the title itself or from somewhere in the text?

Reply:
I'm thinking about adding something like "See :ref:\`shutil-archiving-example-with-basedir\` for how to use \*base_dir\* and \*root_dir\* together." after https://github.com/python/cpython/pull/10367/files#diff-bda6196613f849e0aa43c7e2dd9f6068R540

What do you think?

Reply:
That would be really nice actually. This way someone, who might not scroll all the way down, will not miss the example.

Reply:
Sounds good to me!